### PR TITLE
Added song selection function

### PR DESCRIPTION
**Pull request by urizakii**
Added new function **`song_selection`** to the main program. The function allows the user to select a song from a list of available songs from the available years. The list of songs is stored in the **`songList.json`**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -26,10 +49,87 @@
...
+
+# SONG SELECTION
+def song_selection():
+    global year_choice
+    global song_choice
+
+    while True:
+        print(" == YEAR == ")
+        # counts how many player chose
+        print("SONG CART: ", str(len(songChoice)), "\n")
```
Comment (IwissManalo) [inline main.py]: Wrong naming convention (songChoice), should be (song_choice)😊

File: main.py [hunk 2]
```diff
@@ -26,10 +49,87 @@
...
+        # clears the choices from songs
+        choices.clear()
```
Comment (IwissManalo) [inline main.py]: Wrong variable name (choices), should be (choice)😊

File: main.py [hunk 3]
```diff
@@ -2,18 +2,41 @@
...
+import os
+import json
+
+# clears the terminal using os command
+def clear_screen():
+    os.system('cls')
+
+# reading from the song list json file
+with open('songList.json') as f:
+    songList = json.load(f)
```
Comment (IwissManalo) [inline main.py]: Wrong naming convention (songList), should be (song_list)😊

**Review by IwissManalo — CHANGES_REQUESTED** [review]
Check corrections for lines (15, 75, and 77)😊 Thanky!

**Review by urizakii — COMMENTED** [review]
Issues Resolved.

**Review by IwissManalo — CHANGES_REQUESTED** [review]
Repeated code between lines 75-79

**Review by IwissManalo — APPROVED** [review]
LGTM🤩

**Pull request merged**